evm/hyperliquid.js

```diff
@@ -0,0 +1,225 @@
+
+    const currentDeployer = await token.deployer();
+    printInfo('Current deployer', currentDeployer);
+    printInfo('New deployer', deployer);
+
+    const serviceOwner = await service.owner();
+    const isOperator = await service.isOperator(wallet.address);
+
+    if (wallet.address.toLowerCase() !== serviceOwner.toLowerCase() && !isOperator) {
+        throw new Error('Wallet does not have permission to update deployers. Must be service owner or operator.');
+    }
+
+    const gasOptions = await getGasOptions(chain, options, 'HyperliquidInterchainTokenService');
+    const tx = await service.updateTokenDeployer(tokenId, deployer, gasOptions);
+    await handleTx(tx, chain, service, 'updateTokenDeployer');
+
+    const updatedDeployer = await token.deployer();
+    printInfo('Updated deployer', updatedDeployer);
+}
+
+async function main(processor, args, options) {
+    options.args = args;
+    return mainProcessor(options, (config, chain, options) => {
+        if (!isHyperliquidChain(chain)) {
+            throw new Error(`Chain "${chain.name}" is not supported. This script only works on Hyperliquid chains.`);
+        }
+
+        const rpc = chain.rpc;
+        const provider = getDefaultProvider(rpc);
+        const wallet = new Wallet(options.privateKey, provider);
+
+        return processor(wallet, chain, options);
+    });
+}
+
+async function switchHyperliquidBlockSize(options, useBigBlocks, chain) {
+    const blockType = useBigBlocks ? 'big' : 'small';
+    options.args = [blockType];
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+    const wallet = new Wallet(options.privateKey, provider);
+
+    try {
+        await updateBlockSize(wallet, chain, options);
+    } catch (error) {
+        throw error;
+    }
+}
+
+function shouldUseBigBlocks(key) {
+    return key === 'implementation' || key === 'interchainTokenFactoryImplementation';
+}
+
+if (require.main === module) {
+    const program = new Command();
+
+    program.name('hyperliquid').description('Hyperliquid chain management commands');
+
+    program
+        .command('update-block-size <blockSize>')
+        .description('Update Hyperliquid block size')
+        .action((blockSize, options) => {
+            main(updateBlockSize, [blockSize], options);
+        });
+
+    program
+        .command('deployer <tokenId>')
+        .description('Get deployer address for a Hyperliquid interchain token')
+        .action((tokenId, options) => {
+            main(deployer, [tokenId], options);
+        });
+
+    program
+        .command('update-token-deployer <tokenId> <deployer>')
+        .description('Update deployer address for a Hyperliquid interchain token')
+        .action((tokenId, deployer, options) => {
+            main(updateTokenDeployer, [tokenId, deployer], options);
+        });
+
+    addOptionsToCommands(program, addEvmOptions, { privateKey: true });
```

Comment:
```suggestion
    addOptionsToCommands(program, addEvmOptions);
```
why is this needed? It's already added by default, and not a valid option in the helper